README.md

```diff
@@ -1,6 +1,10 @@
 PEG.js
 ======
 
+[![npm version][npm-badge]][npm-url]
+[![Build Status][travis-badge]][travis-url]
+[![Coverage Status][coveralls-badge]][coveralls-status]
```

Comment:
I don't think the `[coveralls-status]` bit renders properly (see [here](https://github.com/AdrieanKhisbe/pegjs/blob/0536e2c/README.md)). It's supposed to be `[coveralls-url]` instead?